Test Runner/UI Tests/AttachmentCameraControllerTests.swift

```diff
@@ -34,6 +34,8 @@ final class AttachmentCameraControllerTests: XCTestCase {
         let cameraModeLabel = app.staticTexts["Camera Capture Mode"]
         let device = UIDevice.current.userInterfaceIdiom
         let orientation = app.staticTexts["Device Orientation"]
+        let springboard = XCUIApplication(bundleIdentifier: .springboardIdentifier)
```

Comment:
Since, it is used only on 1 place.
```suggestion
        let springboard = XCUIApplication(bundleIdentifier: "com.apple.springboard")
```